.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
     rev: v1.1.0
     hooks:
       - id: detect-secrets
-        args: [ '--baseline', '.secrets.baseline', '--exclude-files', 'tests/*/cassettes/*' ]
```

Comment:
exclude files is still repeated here :(

Reply:
Gotchaaa! It's on purpose! Thanks!

Reply:
Why do you want to exclude dashboard notebooks from secretes detection?